src/IssueViz.Security/IssueViz.Security.csproj

```diff
@@ -32,6 +32,17 @@
 
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Are all of these changes necessary?

It looks like you've added a copy of the `sonarlint.png` file. If so, is it necessary to change the references to the existing file?

Reply:
those lines were added by vs automatically. When i tried to remove those in cleanup Logo was not visible on the menu.